src/site/content/en/blog/cls-web-tooling/index.md

```diff
@@ -0,0 +1,278 @@
+    AND form_factor.name = 'phone')
+SELECT
+  cls.start AS start,
+  cls.`END` AS `end`,
+  cls.density AS cls_density,
+  uncapped_cls.density AS uncapped_cls_density
+FROM
+  new_data
+INNER JOIN
+  old_data
+ON
+  new_data.cls.start = old_data.uncapped_cls.start
+```
+
+You will be able to continue relying on this data to be present for a period of up to 6 months,
+with uncapped CLS being retired on December 14th, 2021.
+
+## Updating the CLS weighting in Lighthouse
+
+When CLS was first introduced in Lighthouse,
+it was a brand new sparkly metric.
+As such, in order to make sure developers had time to test, benchmark,
+and optimize against it,in the performance score CLS was weighed less heavily.
+
+Now, after having had some time in the hands of developers,
+the Lighthouse score has increased the weight of CLS from 5% to 15%,
+consistent with the methodology of having Core Web Vitals
+be the most heavily weighted metrics in the Lighthouse score.
+
+You can find the updated weightings of metrics in Lighthouse v8 in the
+[scoring calculator](https://googlechrome.github.io/lighthouse/scorecalc/).
+
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/UAslFmRMON2y5qtY1TZE.png",
+alt="Lighthouse scoring calculator",
+width="800", height="405" %}
+
+Lighthouse 8.0's CLS implementation includes both windowing and CLS contribution from subframes. Prior to 8.0, CLS in Lighthouse didn't include subframes' CLS in the metric calculation, but now does. And just for confirmation, field CLS measured by CrUX handles also handles windowing and subframes similarly. 
+
+Still, the primary difference between lab and field CLS is that lab CLS's window of observation ends at "fully loaded" as determined under lab conditions, whereas in the field, the window of observation extends to the entire page lifetime, including post-load activity. That said, the windowing adjustment does mitigate this difference substantially.
+
+## Measuring in the field yourself
+
+Should you wish to measure the latest CLS implementation,
+you can also record this for your field data via RUM using the following PerformanceObserver snippet:
+
+```js
+{
+  let max = 0;
+  let curr = 0;
+  let firstTs = Number.NEGATIVE_INFINITY;
+  let prevTs = Number.NEGATIVE_INFINITY;
+
+  new PerformanceObserver((entryList) => {
+    for (const entry of entryList.getEntries()) {
+      if (entry.hadRecentInput) continue;
+      if (entry.startTime - firstTs > 5000 || entry.startTime - prevTs > 1000) {
+        firstTs = entry.startTime;
+        curr = 0;
+      }
+      prevTs = entry.startTime;
+      curr += entry.value;
+      max = Math.max(max, curr);
+      console.log('Current MAX-session-gap1s-limit5s value:', max, entry);
+    }
+  }).observe({type: 'layout-shift', buffered: true});
+}
+```
+
+Or by relying directly on the
+[Web Vitals JavaScript library](https://github.com/GoogleChrome/web-vitals),
+which has also been updated with this change.
+
+## Additional updates
```

Comment:
Super nit: it's fine to include these here if we don't have a better place to post them at the moment, but it feels a bit weird to mention LCP and FCP updates in a post about CLS.

Reply:
I think this was my suggestion, for lack of a better place. Agreed it's not ideal but maybe the best we can do.